docs/vocascan-server/configuration.md

```diff
@@ -109,13 +109,15 @@ For more information about the logger section see [logging guide](vocascan-serve
-| `location` |         | Either the path to your html file, or an internet address for the redirect                                                                                                                                                                              |
-| `langs`    |         | Option to define multiple languages for your static file or redirect. Add props, named after the country code (ISO-639-1: Germany -> de), which include a `type` and a `location`                                                                       |
+| Name                      | Default | Description                                                                                                                                                                                                                                             |
+| ------------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `url`                     |         | The route on which your page will be displayed relative to `/`                                                                                                                                                                                          |
+| `fallback`                |         | Defines a fallback page, if neither a language is specified per query parameter (`?lang=`), nor the queried language exists. You can either set a static page or a redirect to one of the defined languages (recommended if you use multiple languages) |
+| `fallback.type`           |         | Defines whether the server renders a static page or just redirects to another. Options: `file` or `redirect`                                                                                                                                            |
+| `fallback.location`       |         | Either the path to your html file, or an internet address for the redirect                                                                                                                                                                              |
+| `langs`                   |         | Option to define multiple languages for your static file or redirect. Add props, named after the country code ([ISO-639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes): Germany -> de)                                                       |
+| `langs.ISO_LANG.type`     |         | Defines whether the server renders a static page or just redirects to another. Options: `file` or `redirect`                                                                                                                                            |
```

Comment:
I suggest adding something like this to indicate that ISO_LANG is a placeholder. You can decide if you want to use <> or [].
```suggestion
| `langs.<ISO_LANG>.type`     |         | Defines whether the server renders a static page or just redirects to another. Options: `file` or `redirect`                                                                                                                                            |
```